lib/web_ui/lib/src/engine/skwasm/skwasm_impl/shaders.dart

```diff
@@ -33,6 +33,11 @@ class SkwasmGradient extends SkwasmNativeShader implements ui.Gradient {
     ui.TileMode tileMode = ui.TileMode.clamp,
     Float32List? matrix4,
   }) => withStackScope((StackScope scope) {
+    assert(() {
+      validateColorStops(colors, colorStops);
+      return true;
+    }());
```

Comment:
Shouldn't these throw at runtime too? What happens with "wrong" color stops in release?

Reply:
I'm matching the behavior of CanvasKit here, we only validate in debug mode. On release, I guess everything will explode, but that's okay, it's release.